components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -119,10 +119,10 @@ func (r *FunctionReconciler) deploymentSelectorLabels(instance *serverlessv1alph
 }
 
 func (r *FunctionReconciler) podLabels(instance *serverlessv1alpha2.Function) map[string]string {
```

Comment:
This function is unused - is used only by test

Reply:
Removed unused functions.